### ISSUE TITLE
WRF installation seems to hang at "Testing: t_cache_image"

### ISSUE THREAD

**Issue by beuraieon-a** [HathewayWill/WRF-MOSIT#39]
Hello. I am trying to install WRF on a newly installed Linux Mint 22 Cinnamon. In this installation, I chose GNU compiler, OpenGrADS, "Yes" for auto-configuration, "No" for the DTC MET Tools, "Yes" for the WPS Geographical Input Data for Specific Applications, and "No" for the GEOG Optional WPS Geographical Input Data.

The installation process is currently ongoing for the testing of HDF5, but it seems to have hung after printing the line "Testing: t_cache_image". Nearly an hour has already passed and the installation process has not yet progressed. I'm wondering if this is normal, or if there is something I need to fix.

Attached herewith is the "WRF_MOSIT.log" file for your perusal. Thank you very much.

[WRF_MOSIT.log](https://github.com/user-attachments/files/17372914/WRF_MOSIT.log)

![Screenshot from 2024-10-15 13-10-55](https://github.com/user-attachments/assets/154a4b36-e7c5-4a3b-8ee7-35d4d811a808)

**Comment by beuraieon-a** [comment]
I ran the installation script again with the same settings, but the same problem persists. The installation hangs after the line "Testing: t_cache_image".

[WRF_MOSIT.log](https://github.com/user-attachments/files/17374436/WRF_MOSIT.log)

**Comment by HathewayWill** [comment]
I've seen this error before, it's not an error that hdf5 test just takes a very long time when running with limited cpus.  How many cpus does that virtual machine have?

**Comment by beuraieon-a** [comment]
I'm not using a virtual machine. The Linux Mint 22 OS is directly installed in an external hard drive, I just change the booting priority so that I can change OS from the main OS (Windows in my laptop's internal drive) and the Linux OS in the external drive. My laptop has an Intel Core i5-4200U processor.

**Comment by HathewayWill** [comment]
> I'm not using a virtual machine. The Linux Mint 22 OS is directly installed in an external hard drive, I just change the booting priority so that I can change OS from the main OS (Windows in my laptop's internal drive) and the Linux OS in the external drive. My laptop has an Intel Core i5-4200U processor.



Okay so that CPU has 2 cores 4 threads so it is using 2 threads to do that test.  It will take a long time for it.  My first reccoenndation would be to start it up and let it run and see if it ever gets past it.

You can check the processes in Linux in the system monitor.  If it is running still then it's working but it's just taking a long time.

**Comment by beuraieon-a** [comment]
Okay, I'm currently rerunning the installing script and reached the "Testing: t_cache_image" portion. Here's what my System Monitor is showing right now:
![image](https://github.com/user-attachments/assets/54234837-c7fb-449e-a1eb-ade6259eca53)

**Comment by HathewayWill** [comment]
> Okay, I'm currently rerunning the installation script and reached the "Testing: t_cache_image" portion. Here's what my System Monitor is showing right now: ![image](https://private-user-images.githubusercontent.com/155237392/377340972-54234837-c7fb-449e-a1eb-ade6259eca53.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkyMTcxOTksIm5iZiI6MTcyOTIxNjg5OSwicGF0aCI6Ii8xNTUyMzczOTIvMzc3MzQwOTcyLTU0MjM0ODM3LWM3ZmItNDQ5ZS1hMWViLWFkZTYyNTllY2E1My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxOFQwMjAxMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NDNiZTgyNGVhYTYwOWZiY2EyZDNjYjViMzY1NGQ1NjdlMDRiNjA3ZTg4ZWNhMWRhZGVlNDRmY2MxYTU4NGRhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.geMyxF81SGqgkdCXRRzrmc3dAlS3i1rmMhPRzGRbB98)

Yes it is running based on the orange bar at the top being maxed

**Comment by beuraieon-a** [comment]
Okay, so it just takes a really long time for it to finish. I started the installation at around 2:00 PM and the installation got stuck at "Testing: t_cache_image" for nearly 3 hours, I was forced to terminate the process (since I'm doing the installation at our college's computer laboratory, which closes at 5:00 PM). I think I'll just try the installation again some other time. Thank you very much for notifying me.

I'm wondering, though, why is it very slow now? Because I had used the same installation script before (for WRF v.4.5.2) for the same laptop and OS. That time, the process passed through all the needed library installations rather smoothly (albeit with some technical issues) and I was able to install WRF around 2 or 3 hours after starting, and the process started lagging only by the time it began downloading the static geographic data (due to Internet speed). Is this somehow related to the storage space? Since that time my storage was nearly 400 GB, but I recently partitioned my external hard drive into halves so that I can use ~200 GB for file storage, while the other half for the Linux Mint. Thus, I tweaked the installtion script a bit so that it skips the test for storage space. Did that perhaps caused this lagging of the installation process at HDF5?
